lib/.gitignore

```diff
@@ -0,0 +1 @@
+massager_parameters_ex
```

Comment:
I don't want the non-template parameters file to get checked into git inadvertantly (as it would have customer-specific information).